src/portuguese.js

```diff
@@ -0,0 +1,43 @@
+
+class PresenteIndicativo {
+  conjugate (word, info) {
+    let ending = [];
+    switch (word.slice(-2)) {
+      case 'ar':
+        ending = info.singular ? ['o', 'as', 'a'] : ending = ['amos', 'ais', 'am'];
```

Comment:
`endings = info.singular ? ['o', 'as', 'a'] : ['amos', 'ais', 'am'];`
Since it is an array, `endings` is a more appropriate variable name.